### PR TITLE
Add sax test on public_transport tag

### PR DESCRIPTION
**Pull request by nlehuby**

### CODE REVIEW

File: plugins/Public_Transport_Stop.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    def node(self, data, tags):
+        err = []
+        if u"public_transport" not in tags:
+            if "highway" in tags and tags["highway"] == "bus_stop":
+                err.append({"class": 21411, "subclass": 0,
+                            "text": T_("Bus stop without public_transport tag"),
+                            "fix": {"public_transport": "platform"}
+                            })
+            if "railway" in tags and tags["railway"] == "tram_stop":
+                err.append({"class": 21411, "subclass": 0,
```
Comment (frodrigo) [inline plugins/Public_Transport_Stop.py]: You can set subclass to 1 here to distinguate with previous one.

File: plugins/Public_Transport_Stop.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+###########################################################################
+
+from plugins.Plugin import Plugin
+
+
+class Public_Transport_Stop(Plugin):
+
+    def init(self, logger):
+        Plugin.init(self, logger)
+        self.errors[21411] = {"item": 2141, "level": 3, "tag": ["tag", "public_transport", "fix:chair"], "desc": T_(
```
Comment (frodrigo) [inline plugins/Public_Transport_Stop.py]: We can stay with 2140 here. We try to group classes on menu items on the front.

**Comment by nlehuby** [comment]
I still mix up class, subclass and item :confused:

**Comment by frodrigo** [comment]
For me also it was not so easy ;-).

Categories are group of items.
Categories and items are for display and filter purpose on the frontend.

Subclass are subelement of class (yes really).
Class and subclass are for distinguish the analyses results.

Class are set into item by thematic, one anayser can produce different classes set in different items.

**Comment by frodrigo** [comment]
@jocelynj PR good for me

**Comment by jocelynj** [comment]
Thanks for the merge request - I've just merged it to master.

**Pull request merged**